tex/gregoriotex-main.tex

```diff
@@ -832,77 +779,14 @@
     ]%
 }
 
-%% macro that draws the stafflines on the first line, it is different from others due to the initial that can take some place, without lines
-\def\gre@drawfirstlines{%
-  \gre@trace{gre@drawfirstlines}%
-  \advance\gre@dimen@stafflinewidth by -\gre@dimen@initialwidth\relax%
-  %\advance\gre@dimen@stafflinewidth by -\gre@space@dimen@minimalspaceatlinebeginning
-  %\gre@dimen@initialwidth=0pt
-  \hbox to 0pt{%
-    \vbox{%
-      \gre@style@normalstafflines%
```

Comment:
There wasn't `\vskip\gre@space@dimen@spaceabovelines` here on the first line, but there is on subsequent lines. The PR makes the printing of the first line the same as the subsequent lines, so it adds `\vskip\gre@space@dimen@spaceabovelines` and I'm not sure if that's a bug.

Reply:
@rpspringuel Thanks for merging! Did you see this comment -- it will be an easy fix if you want to preserve the old spacing exactly.

Reply:
I'm going to leave it as is for now.  Adding that space above the first line makes sense to me.  We'll see if anyone was relying on the old behavior and complains.

Reply:
OK, great. It seems to be what the docs imply as well.